### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by unional**
I clean up some loose ends for the project.

Hope you will like it. 🌷

### CODE REVIEW

**Comment by unional** [comment]
No problem. The only things es6 in there is `var` vs `const` and string literals.

I can remove that.

**Comment by KoryNunn** [comment]
Also use Strick, I'm happy to be convinced that it is useful, but I never use it and I've never noticed issues.

**Comment by unional** [comment]
yes, also removed. Added to enabled es6 on node.

File: index.js [hunk 1]
```diff
@@ -2,30 +2,35 @@
...
 globalify({
         module: moduleName,
         version: version,
         globalVariable: program.globalVariable,
-        instalDirectory: packageJson.instalDirectory
+        installDirectory: packageJson.installDirectory
     },
-    function(error){
-        if(error){
+    function (error) {
```
Comment (KoryNunn) [inline index.js]: Please change back to the existing style

Comment (unional) [inline index.js]: ok

**Comment by KoryNunn** [comment]
Happy with the rest tho. :+1:

File: index.js (hunk 1 above)
Comment (unional) [inline index.js]: Updated

**Pull request merged**